node/_tools/test/parallel/test-fs-read-stream-pos.js

```diff
@@ -37,6 +39,7 @@ let stream;
 
 const readInterval = setInterval(() => {
   if (stream) return;
+  if (!appended) return;
 
   stream = fs.createReadStream(file, {
     highWaterMark: hwm,
```

Comment:
This test was failing about 10 / 30 of the time, but it seems to be a false positive.
If `createReadStream()` is executed before the append to the test file is completed, the "data" event will not be called and `common.mustCallAtLeast()` verification will fail. 
It is most likely an I/O performance issue, not ReadStream. The performance of `fs.writeFileSync()` in Deno is about 5 or 6 times slower than in Node. (Confirmed with Deno 1.25.3 and Node.js v18.9.0 on my laptop)

To avoid the issue, I have modified the test to make sure that the append to the test file is complete before executing `createReadStream()`.